### PR TITLE
Users/laramume/addv2.1 swagger

### PR DESCRIPTION
**Pull request by laramume**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json [hunk 1]
```diff
@@ -0,0 +1,644 @@
...
+    ]
+  },
+    "paths": {
+        "/languages": {
+            "post": {
+              "tags": [
+                "Languages"
+              ],
+                "description": "The API returns the detected language and a numeric score between 0 and 1. Scores close to 1 indicate 100% certainty that the identified language is true. A total of 120 languages are supported.",
+                "operationId": "56f30ceeeda5650db055a3c7",
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: `operationId` should "{operationGroupName}_{operationName}" better keep consistent with previous versions.

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json [hunk 2]
```diff
@@ -0,0 +1,644 @@
...
+            }
+        },
+        "/entities": {
+            "post": {
+              "tags": [
+                "Entities"
+              ],
+                "description": "The API returns a list of known entities and general named entities (\\\"Person\\\", \\\"Location\\\", \\\"Organization\\\" etc) in a given document. Known entities are returned with Wikipedia Id and Wikipedia link, and also Bing Id which can be used in Bing Entity Search API. General named entities are returned with entity types. If a general named entity is also a known entity, then all information regarding it (Wikipedia Id, Bing Id, entity type etc) will be returned. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/how-tos/text-analytics-how-to-entity-linking#supported-types-for-named-entity-recognition\">Supported Entity Types in Text Analytics API</a> for the list of supported Entity Types. See the <a href=\"https://docs.microsoft.com/en-us/azure/cognitive-services/text-analytics/text-analytics-supported-languages\">Supported languages in Text Analytics API</a> for the list of enabled languages.\n",
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: `\\\"Organization\\\"`, you can just use
```
`Organization`
```
Easier to read as json and better display in auto-generated documents.

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json [hunk 3]
```diff
@@ -0,0 +1,644 @@
...
+                    "default": {
+                        "description": "Error Response",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                },
+                "x-ms-examples": {
+                    "Successful Detect Language request": {
+                    "$ref": "./examples//SuccessfulEntitiesRequest.json"
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: `/`
There are a few more

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json [hunk 4]
```diff
@@ -0,0 +1,644 @@
...
+                    "description": "Errors and Warnings by document",
+                    "type": "array",
+                    "items": {
+                        "$ref": "#/definitions/ErrorRecord"
+                    },
+                    "readOnly": true
+                },
+                "statistics": {
+                    "$ref": "#/definitions/RequestStatistics",
+                    "description": "/// <summary>\r\n            (Optional) if showStats=true was specified in the request this field will contain information about the \r\n            request payload.\r\n            </summary>",
```
Comment (yangyuan) [inline specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: should be markdown or html

**Review by yangyuan — CHANGES_REQUESTED** [review]

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json (hunk 1 above)
Comment (laramume) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: Made it consistent with previous versions

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json (hunk 2 above)
Comment (laramume) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: Updated the description to be consistent with previous versions

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json (hunk 4 above)
Comment (laramume) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: Modified it.

File: specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json (hunk 3 above)
Comment (laramume) [reply specification/cognitiveservices/data-plane/TextAnalytics/stable/v2.1/TextAnalytics.json]: Fixed it.

**Review by yangyuan — APPROVED** [review]

**Review by anuchandy — APPROVED** [review]

**Comment by anuchandy** [comment]
Synced with @laramume, deployment for this api version is started and available in some regions. By early next week should be there in all regions.

**Pull request merged**